src/runtime/Native/NativeTypeSpec.cs

```diff
@@ -10,7 +10,7 @@ struct NativeTypeSpec : IDisposable
         public readonly StrPtr Name;
         public readonly int BasicSize;
         public readonly int ItemSize;
-        public readonly TypeFlags Flags;
+        public readonly IntPtr Flags;
```

Comment:
Hmm, `Flags` is clearly defined as a C `int`, which should "always" be 32bit, right? (https://docs.python.org/3/c-api/type.html#c.PyType_Spec)